.github/workflows/client.yml

```diff
@@ -121,10 +124,15 @@ jobs:
         env:
           IMAGE_NAME: "keep-client"
         with:
+          target: runtime-docker
           # GCR image should be named according to following convention:
           # HOSTNAME/PROJECT-ID/IMAGE:TAG
           # We don't use TAG yet, will be added at later stages of work on RFC-18.
```

Comment:
This comment can be removed now.

Reply:
aba24071481296aeb4aa3e3547eb905d82f31c95